graphql/src/schema/ast.rs

```diff
@@ -25,14 +25,14 @@ pub(crate) fn parse_field_as_filter(key: &Name) -> (Name, FilterOp) {
         k if k.ends_with("_lt") => ("_lt", FilterOp::LessThan),
```

Comment:
This file doesn't seem to have changed other than reordering the match branches. Is there a reason for changing the order?

Reply:
The order is important here because some of the arms' match strings are contained in others, so the previous order created essentially unreachable arms.  Example: if the .ends_with("_in") arm comes before .ends_with("_not_in") the "_not_in" arm will never be matched because "_not_in" ends_with "_in".         

For this reason I made sure:
-  "_not_starts_with" comes before "_starts_with"
-  "_not_ends_with" comes before "_ends_with" 
- "_in" comes before "_not_in"
-  "_contains" comes before "_not_contains".

Reply:
Oh, _of course_! Thanks for explaining, I should've seen that...